Endpoints is_ready() fails on ready endpoint

The Endpoints is_ready method fails if the endpoints subsets does not contain any not_ready_addresses.

For example:
```        # check if any of the endpoint subsets has and addresses
        # in the not ready state.
        for subset in self.obj.subsets:
>           if subset.not_ready_addresses > 0:
E           TypeError: '>' not supported between instances of 'NoneType' and 'int'

/usr/local/lib/python3.6/dist-packages/kubetest/objects/endpoints.py:96: TypeError```

--- a/kubetest/objects/endpoints.py
+++ b/kubetest/objects/endpoints.py
@@ -93,6 +93,6 @@ class Endpoints(ApiObject):
         # check if any of the endpoint subsets has and addresses
         # in the not ready state.
         for subset in self.obj.subsets:
-            if subset.not_ready_addresses > 0:
+            if subset.not_ready_addresses is not None and subset.not_ready_addresses > 0:
                 return False
         return True

--- a/kubetest/objects/endpoints.py
+++ b/kubetest/objects/endpoints.py
@@ -92,7 +92,10 @@ class Endpoints(ApiObject):
 
         # check if any of the endpoint subsets has and addresses
         # in the not ready state.
+        if self.obj.subsets is None:
+            return False
+
         for subset in self.obj.subsets:
-            if subset.not_ready_addresses is not None and subset.not_ready_addresses > 0:
+            if subset.not_ready_addresses is not None and len(subset.not_ready_addresses) > 0:
                 return False
         return True

